src/components/summary/_macro.njk

```diff
@@ -120,7 +120,7 @@
                     {% endif %}
 
                     {% if group.summaryLink is defined and group.summaryLink %}
-                        <div class="summary__add-link{% if group.placeholderText is not defined and group.rows is defined and group.rows | length > 1 %} u-bt{% endif %}">
+                        <div class="{% if group.placeholderText is defined and group.placeholderText or group.rows is defined and group.rows %}summary__add-link{% endif %}{% if group.placeholderText is not defined and group.rows is defined and group.rows | length > 1 %} u-bt{% endif %}">
```

Comment:
Census WLH summary doesn't require `rows` or `placeholderText` but still requires the add link so not sure this is right?